src/features/account/components/AccountInfo.tsx

```diff
@@ -0,0 +1,43 @@
+import { CircularProgress, Heading, VStack } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+import { useEnsName } from 'wagmi';
+
+import { AccountRoleTags, AccountRole } from '.';
+import { JazzIcon } from '../../../components/JazzIcon';
```

Comment:
If you export this in `components/index.ts`, this import can be cleaner.

Reply:
noted

Reply:
updated